Kitodo-LongTimePreservationValidationModule/src/main/java/org/kitodo/longtimepreservationvalidationmodule/KitodoOutputHandler.java

```diff
@@ -0,0 +1,160 @@
+     *
+     * @return {@code null}. The value is not used.
+     */
+    private static final String beforeParentConstructor() {
+        Logger.getLogger("edu.harvard.hul.ois.jhove.handler").setLevel(Level.OFF);
+        return null;
+    }
+
+    private final List<String> messages = new ArrayList<String>();
+
+    private ValidationResultState state;
+
+    protected KitodoOutputHandler() {
+        super(beforeParentConstructor(), null, new int[] {1970, 1, 1 }, null, null);
+    }
+
+    private void addMessage(Message message) {
+        StringBuilder line = new StringBuilder(127);
+        line.append(message.getMessage());
+        String subMessage = message.getSubMessage();
+        if (subMessage != null) {
+            line.append(": ");
+            line.append(subMessage);
+        }
+        messages.add(line.toString());
+        long offset = message.getOffset();
+        if (offset > -1) {
+            messages.add("Offset: " + offset);
+        }
+    }
+
+    @Override
+    public void show() {
+    }
+
+    @Override
+    public void show(App app) {
+    }
+
+    @Override
+    public void show(Module module) {
+    }
+
+    @Override
+    public void show(OutputHandler handler) {
+    }
+
+    @Override
+    public void show(RepInfo info) {
+        state = ValidationResultState.valueOf(info.getWellFormed(),
+            super._je.getSignatureFlag() ? RepInfo.UNDETERMINED : info.getValid());
+        info.getMessage().forEach(x -> addMessage(x));
+    }
+
+    @Override
+    public void showFooter() {
+    }
+
+    @Override
+    public void showHeader() {
+    }
+
+    /**
+     * Returns a concise textual representation of this Kitodo output handler.
+     * Will be called from debuggers.
+     *
+     * @return representation of this
+     */
+    @Override
+    public String toString() {
+        return messages.stream().collect(Collectors.joining(", ", String.valueOf(state).concat(" ("), ")"));
+    }
+
+    /**
+     * Returns the captured results as validation result. The function uses
+     * reflection since the result’s constructor is package private.
+     *
+     * @return the created validation result
+     */
+    public ValidationResult toValidationResult() {
+        try {
+            return VALIDATION_RESULT_CONSTRUCTOR.newInstance(state.toState(), messages);
+        } catch (InstantiationException | IllegalAccessException | InvocationTargetException e) {
+            throw new UndeclaredThrowableException(e);
+        }
+    }
+
+    /**
+     * Sets the state to ERROR and the exception as message, including the stack
+     * trace.
+     *
+     * @param e
+     *            exception to set
+     */
+    public void treatException(Exception e) {
+        state = ValidationResultState.ERROR;
+        if (e.getMessage() != null) {
+            messages.add(e.getMessage());
+        }
+        messages.add(e.getClass().getSimpleName());
+        StringWriter buffer = new StringWriter();
```

Comment:
Opening a stream without closing it should be avoided. Please use a try-with-resources statement here.